src/javascript/app/pages/trade/tick_trade.js

```diff
@@ -337,6 +337,10 @@ const TickDisplay = (() => {
     const dispatch = (data) => {
         const tick_chart = CommonFunctions.getElementById(id_render);
 
+        if (typeof applicable_ticks === 'undefined') {
+            applicable_ticks = [];
```

Comment:
why not just initiate it as an empty array on definition?

Reply:
currently it is initialized here: https://github.com/binary-com/binary-static/blob/master/src/javascript/app/pages/trade/tick_trade.js#L334, probably we can init it here as well: https://github.com/binary-com/binary-static/blob/master/src/javascript/app/pages/trade/tick_trade.js#L56 ?